config/field_test.yml

```diff
@@ -0,0 +1,25 @@
+experiments:
+  user_home_feed: # Home feed collection for logged in user
+    variants:
+      - base
+      - more_random
+      - top_past_week_random
+      - newer
+    weights:
+      - 25
```

Comment:
The gem expects us to perhaps "declare a winner", but I think I'd rather this be more of a rolling thing and we can swap variants on the fly a bit and move weights around rather than the binary win/loss.

I feel like we could continuously delete old field test memberships and results so that we can keep funneling users back through the experiment.

Win/loss binary might make more sense for more isolated tests.

Reply:
Hmm..I guess that depends if we view these experiments are being long lived or not. I can see home page feed experiments lasting a long time.

For shorter lived experiments, I'd argue you don't want it to sit around in FieldTest for any length of time: once you have a winner, yank the A/B test code around it!